README.md

```diff
@@ -0,0 +1,98 @@
+
+## How it works
+
+tox creates virtual environments for all configured so called `testenvs`, it then installs the project and other necessary dependencies and runs the configured set of commands.
+
+![tox flow](https://tox.readthedocs.io/en/latest/_images/tox_flow.png)
+
+See [system overview](https://tox.readthedocs.io/en/latest/#system-overview) for more details.
+
+## tox can be used for ...
+
+* creating develepment environments
+* running static code analysis and test tools
+* automating package builds
+* running tests against the package build by tox
+* checking that packages install correctly with different Python versions/interpreters
+* unifying Continuous Integration and command line based testing
+* building and deploying project documentation
+* release automation
+* limit: your imagination
+
+## Usage
+
+tox is mainly used as a command line tool and needs a `tox.ini` or a `tool.tox`
+section in `pyproject.toml` containing the configuration.
+
+### A simple example
+
+To test a simple project that has some tests, here is an example with the `tox.ini`
+in the root of the project:
+
+```ini
+[tox]
+envlist = py27,py36
+
+[testenv]
+deps = pytest       # install pytest in the virtualenv where commands will be executed
+commands =
+    # whatever extra steps before testing might be necessary
+    pytest          # or any other test runner that you might use
+```
+
+```console
+$ tox
+
+[lots of output from what tox does]
+[lots of output from commands that were run]
+
+__________________ summary _________________
+  py27: commands succeeded
+  py36: commands succeeded
+  congratulations :)
+```
+
+tox created two `testenvs` - one based on Python2.7 and one based on Python3.6, it installed pytest in them and ran the tests. The report at the end summarizes which
+`testenvs` have failed and which have succeeded.
+
+## Projects using tox
+
+tox is widely used, so this is only a very small selection:
+
+* [ansible](https://github.com/ansible/ansible)
+* [devpi](https://github.com/devpi) (devpi and plugins/tools)
+* [django](https://github.com/django/django)
+* [httpie](https://github.com/jakubroztocil/httpie)
+* [pallets](https://github.com/pallets) (flask, jinja, etc)
+* [pandas](https://github.com/pandas-dev/pandas)
+* [pytest-dev](https://github.com/pytest-dev) (pytest and plugins/tools)
+* [requests](https://github.com/requests/requests)
+* [tox-dev](https://github.com/tox-dev) (obviously)
+* [and many more](https://github.com/tox-dev/tox/network/dependents)
+
+---
+
+Contributions are welcome. See [contributing](https://github.com/tox-dev/tox/blob/master/CONTRIBUTING.rst) and our [Contributor Covenant Code of Conduct](https://github.com/tox-dev/tox/blob/master/CODE_OF_CONDUCT.md).
+
+* [docs are here](https://tox.readthedocs.org)
+* [code is here](https://github.com/tox-dev/tox)
+* [issue tracker is here](https://github.com/tox-dev/tox/issues)
+* [license is MIT](https://github.com/tox-dev/tox/blob/master/LICENSE)
```

Comment:
I'd also like to see this whole bottom section changed. I'd say this could be done in a more clean way. See above example for how this can be achieved.